packages/quantic/force-app/main/default/lwc/quanticDateFacet/quanticDateFacet.html

```diff
@@ -1,11 +1,23 @@
 <template>
   <template if:true={hasValues}>
     <lightning-card title={label}>
-      <ul class="slds-has-dividers_around-space slds-m-vertical_medium">
-        <template for:each={values} for:item="v">
-          <c-quantic-date-facet-value onselectvalue={onSelect} key={v.start} item={v} is-checked={v.checked}></c-quantic-date-facet-value>
-        </template>
-      </ul>
+      <button class="slds-button slds-button_icon slds-p-bottom_small facet__collapse" slot="actions" onclick={toggleFacetVisibility}>
+        <lightning-icon class="slds-button__icon" icon-name={facetIconName} size="x-small"></lightning-icon>
+      </button>
```

Comment:
Actually, these exist:
https://developer.salesforce.com/docs/component-library/bundle/lightning-button-icon/example so... 

```suggestion
      <lightning-button-icon icon-name={facetIconName}  alternative-text={something} title="collapse/show facet" variant="bare"></lightning-button-icon>
```

Reply:
That does work. The only thing I was not sure about and it's also the case for using `lightning-button` for the clear button, is that if I use those I can't get rid of the weird blue border that appears around the buttons to indicate that it is in focus:

![Screen Shot 2021-08-09 at 4 50 27 PM](https://user-images.githubusercontent.com/83351444/128772672-9c3792c3-4712-4822-a7b6-0d980cc4b991.png)

It only goes way when some other element gets the focus.

Is that something we care about?

Reply:
I would say no, it does seem like something that would help accessibility? perhaps someone navigating using tab or something? It would show the button highlighted so they can press enter to activate it?

Reply:
I agree, and also, using `lightning-button` has allowed me to remove two css files that were no longer necessary. So, good stuff all around